setup.py

```diff
@@ -19,8 +19,12 @@
     license='BSD',
 
     install_requires=[
-        "fhurl>=0.1.7", "smarturls", "six", "Django>=1.3",
-        "dj-database-url",
+        "beautifulsoup4",
+        "Django>=1.8,<1.9",
```

Comment:
Do we really need >=1.8? Can it be more relaxed?

Also for the rest too, its not recommended to put exact version number in install_requires I think, if someone has Pillow==2.9.1, it won't work, but I am sure it will work with almost any version of Pillow. Remove all versions, put versions only when we know for a fact that it is not supported.